lib/matplotlib/tests/test_axes.py

```diff
@@ -786,6 +786,23 @@ def test_hexbin_log():
     plt.colorbar(h)
 
 
+@image_comparison(['hexbin_linear.png'], style='mpl20')
+def test_hexbin_linear():
+    # Issue #21165 (and also test linear scaled)
+
+    # Remove this line when this test image is regenerated.
+    plt.rcParams['pcolormesh.snap'] = False
```

Comment:
This is a completely new image; you don't need this line.

Reply:
ok, I will solve it, thanks

Reply:
@QuLogic But when I remove it I receive this error message:
```py
===================================================================================== test session starts =====================================================================================
platform linux -- Python 3.10.0, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/lucas/matplotlib, configfile: pytest.ini
collected 1 item                                                                                                                                                                              

test_axes.py F                                                                                                                                                                          [100%]

========================================================================================== FAILURES ===========================================================================================
___________________________________________________________________________________ test_hexbin_linear[png] ___________________________________________________________________________________

args = (), kwds = {'extension': 'png', 'request': <FixtureRequest for <Function test_hexbin_linear[png]>>}

    @wraps(func)
    def inner(*args, **kwds):
        with self._recreate_cm():
>           return func(*args, **kwds)
E           matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 0.101):
E           	result_images/test_axes/hexbin_linear.png
E           	result_images/test_axes/hexbin_linear-expected.png
E           	result_images/test_axes/hexbin_linear-failed-diff.png

/home/lucas/.asdf/installs/python/3.10.0/lib/python3.10/contextlib.py:79: ImageComparisonFailure
=================================================================================== short test summary info ===================================================================================
FAILED test_axes.py::test_hexbin_linear[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 0.101):
====================================================================================== 1 failed in 1.04s ======================================================================================
```

Reply:
nm, I think is a problem in my PC

Reply:
@QuLogic  but when I delete it I receive a error